### PR TITLE
feat: add switchWindow command

### PR DESCRIPTION
**Pull request by magne4000**
This PR adds a `switchWindow` command as documented in [recent versions of WebdriverIO](https://webdriver.io/docs/api/browser/switchWindow.html)

### CODE REVIEW

**Comment by codebytere** [comment]
@magne4000 some conflicts here: could you please rebase and then i can give this a review?

**Comment by magne4000** [comment]
@codebytere here you go :+1:

File: lib/application.js [hunk 1]
```diff
@@ -225,6 +225,40 @@ Application.prototype.addCommands = function () {
     })
   })
 
+  /**
+   * Utility from webdriverio v5
+   * https://github.com/webdriverio/webdriverio/blob/v5.9.4/packages/webdriverio/src/commands/browser/switchWindow.js
+   */
+  this.client.addCommand('switchWindow', async function (urlOrTitleToMatch) {
+    if (typeof urlOrTitleToMatch !== 'string' && !(urlOrTitleToMatch instanceof RegExp)) {
+      throw new Error('Unsupported parameter for switchWindow, required is "string" or a RegExp')
```
Comment (codebytere) [inline lib/application.js]: Let's throw a `TypeError` here and say `Invalid parameter urlOrTitleToMatch: expected a string or a RegExp` for clarity.

**Comment by magne4000** [comment]
@codebytere Everything should be OK now :+1:

**Pull request merged**